typedoc.json

```diff
@@ -3,6 +3,20 @@
         "packages/*"
     ],
     "exclude": [
+        "packages/cjs-module-analyzer",
+        "packages/compartment-mapper",
+        "packages/daemon",
```

Comment:
@kriskowal I'm suggesting / proposing that the daemon package is an implementation detail for the cli package. I'd like to know your take on it. Does that seem right? Or are there docs in the daemon package intended for users / devs?

Reply:
The daemon is definitely an implementation detail of something else and the CLI is the only example so far. However, the daemon does provide the API for caplets.

Reply:
I don't see anything about caplets on https://endojs.github.io/endo/modules/_endo_daemon.html
I guess its `tsdoc.json` needs some love?

Reply:
I don't see docs on an API for caplets in this package.

https://github.com/endojs/endo/tree/master/packages/cli/demo is the only docs I've seen for the endo daemon. I'm struggling to get `typedoc` to include it in the output. @turadg , clues?

Reply:
add an `entrypoint` in the typedoc.json for the package

Reply:
I should have mentioned that I tried that; got:

```
[warning] The entry point ./packages/cli/demo/README.md is not referenced by the 'files' or 'include' option in your tsconfig.
```

tried adding that to the `include` key of `packages/cli/tsconfig.json` but the symptoms persist.

Reply:
More important: Neither daemon nor cli are stable and published, so it would be good to withhold docs for now.